geometry_msgs/Twist velocity
string[] wheel_link_names
float64[] drive_constraint_errors
float64[] longitudinal_slip_constraint_errors